src/envo/envo-edit.owl

```diff
@@ -39742,6 +39744,37 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> <http://purl.obolibrar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000066> <http://purl.obolibrary.org/obo/ENVO_00000015>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_01001497>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105014> (biodegradation)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://doi.org/10.1351%2FPAC-REC-10-12-04>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105014> "A biological process during which organic matter is broken down by microorganisms."@en)
```

Comment:
@meichen-liu would you mind double checking if this has to be limited to organic matter alone? Watch out for the weird semantics of "organic" in chemistry. 

Please also check that this is only done by microorganisms - fungi can biodegrade quite a range of things and often do so in macroscopic form.

Reply:
Thanks @pbuttigieg for pointing that out.

I looked it up and seems like biodegradation is not limited to organic matter. Based on this definition I found

> Degradation caused by enzymatic process resulting from the action of cells.

I re-write the definition of biodegradation as

> A biological process during which substance or chemical is broken down due to the action of cells.